compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -334,6 +334,16 @@ extern "C" void LLVMRustAddStructRetAttr(LLVMValueRef Fn, unsigned Index,
   AddAttribute(F, Index, Attr);
 }
 
+extern "C" void LLVMRustEmitUWTableAttr(LLVMValueRef Fn, bool async) {
```

Comment:
nit: To keep with the general LLVM style, this should be `bool Async`.

Reply:
Fixed. Sigh.